lib/queueLogging.js

```diff
@@ -23,11 +28,67 @@ module.exports = (queue, server) => {
         data: job.data,
         progress: job._progress,
         attempts: job.attempts,
-        attemptsMade: job.attemptsMade,
-      },
+        attemptsMade: job.attemptsMade
+      }
     };
 
     server.log(['bullish', queue.name, 'error'], err);
     server.emit('bullish failed', err);
   });
+
+  queue.on('active', function(job /*jobPromise*/) {
+    const logObj = {
+      queueName: queue.name,
+      job: {
+        jobId: job.jobId,
+        timestamp: job.timestamp,
+        deplay: job.delay,
+        data: job.data,
+        progress: job._progress,
+        attempts: job.attempts,
+        attemptsMade: job.attemptsMade
+      }
+    };
+    server.log(['bullish', queue.name, 'active'], logObj);
```

Comment:
Do you think, all these debug logging is neccessary? Maybe you want to introduce a `debug` option to bullish?

Reply:
It is utilizing hapis internal logging. You can on hapi level control what happens to those logs. You can also choose to ignore them.